src/code/HTTPRequestHandler/httprequesthandlerDELETE.cpp

```diff
@@ -0,0 +1,25 @@
+#include <HTTPRequestHandler/httprequesthandlerDELETE.hpp>
+
+bool httprequesthandlerDELETE::fileExists(const std::string &path)
```

Comment:
since we will have similar logic regarding files in other places, I think that you can create a class / namespace for files like: `fileManager` / `File` / `filer`.

we will use it when loding the conf file and maybe also in GET and POST